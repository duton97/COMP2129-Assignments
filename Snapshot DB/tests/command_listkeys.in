LIST KEYS
SET a 1 2 3
SET b 1 2 3
LIS KEYS
BYE
