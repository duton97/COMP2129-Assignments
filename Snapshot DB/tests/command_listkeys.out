> no keys

> ok

> ok

> b
a

> bye
